<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2997</width>
    <height>2761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>2501</width>
      <height>1751</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Adherents</string>
     </attribute>
     <widget class="QLabel" name="le_id_2">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>-170</y>
        <width>2081</width>
        <height>1561</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/274322707_653768079170100_6282390474491387313_n (2).jpg</pixmap>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>731</width>
        <height>471</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Century Gothic&quot;;

font: 75 16pt &quot;Century Gothic&quot;;

color: rgb(255, 255, 255);

</string>
      </property>
      <property name="title">
       <string>Ajouter Adherents</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>130</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>190</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Prénom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>250</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Métier </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>310</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Sexe</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>242</x>
         <y>121</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(255, 255, 255);

background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom">
       <property name="geometry">
        <rect>
         <x>242</x>
         <y>180</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(255, 255, 255);

background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_metier">
       <property name="geometry">
        <rect>
         <x>242</x>
         <y>241</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(255, 255, 255);

background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>420</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Century Gothic&quot;;

background-color: rgb(226, 27, 27);

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Ajouter Adherents</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>360</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Cotisation</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_sexe">
       <property name="geometry">
        <rect>
         <x>242</x>
         <y>301</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(255, 255, 255);

background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>61</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(255, 255, 255);

background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_cotisation">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>360</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(255, 255, 255);

background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>540</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Downloads/supprimer.png</normaloff>../../Downloads/supprimer.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>70</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="le_modif">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>550</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Downloads/modifier-le-fichier (1).png</normaloff>../../Downloads/modifier-le-fichier (1).png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>690</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Century Gothic&quot;;

background-color: rgb(226, 27, 27);
background-color: rgb(0, 0, 0);

color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Tri</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>760</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Century Gothic&quot;;

background-color: rgb(0, 0, 0);

color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_adherent">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>540</y>
        <width>751</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="le_afficher">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>920</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>donneur</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-160</y>
        <width>2081</width>
        <height>1561</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/274322707_653768079170100_6282390474491387313_n (2).jpg</pixmap>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>741</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Century Gothic&quot;;

font: 75 16pt &quot;Century Gothic&quot;;

color: rgb(255, 255, 255);

</string>
      </property>
      <property name="title">
       <string>Ajouter Benevoles</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>160</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>210</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Métier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>320</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Sexe</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_d">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(255, 255, 255);

background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom_d">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>120</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(255, 255, 255);

background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom_d">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>170</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(255, 255, 255);

background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_metier_d">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>230</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(255, 255, 255);

background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>370</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Century Gothic&quot;;

background-color: rgb(226, 27, 27);

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Ajouter Benevoles</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_sexe_d">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>320</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(255, 255, 255);

background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>520</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Downloads/supprimer.png</normaloff>../../Downloads/supprimer.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>590</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Downloads/modifier-le-fichier.png</normaloff>../../Downloads/modifier-le-fichier.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>670</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Century Gothic&quot;;

background-color: rgb(226, 27, 27);
background-color: rgb(0, 0, 0);

color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Tri</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>730</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Century Gothic&quot;;

background-color: rgb(226, 27, 27);
background-color: rgb(0, 0, 0);

color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QPushButton" name="le_afficher_d">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>860</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>affichier</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_donneur">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>521</y>
        <width>541</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2997</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
